code/game/objects/items/RPD.dm

```diff
@@ -237,10 +237,15 @@ GLOBAL_LIST_INIT(transit_tube_recipes, list(
 	var/constructed = FALSE
 	///Stores the internal tank reference
 	var/obj/item/tank/internal_tank
+	///Differentiate the normal RPD from the borg ones since they don't use pressure
+	var/use_pressure = TRUE
 
 /obj/item/pipe_dispenser/constructed
 	constructed = TRUE
 
+/obj/item/pipe_dispenser/cyborg
+	use_pressure = FALSE
+
```

Comment:
Assuming this is due to refil concerns, you could also have attacking a canister with the RPD work as filling it up (perhaps with a do-after simulating attaching the RPD's quick-refil nozzle to the canister's tank port)

Then the borg RPD could work identically to the crew's version.

Reply:
engineering borgs taking more of our jobs is a bad thing